.github/workflows/publish.yml

```diff
@@ -1,25 +1,25 @@
 name: Publish @m3ter-com/react-pivottable to NPM
 on:
-  push:
-    branches: [master]
+    push:
```

Comment:
Isn't Yaml very strict on spaces? Looks like this had added 2 more in loads of places

Reply:
Ah, yeah, guessing we've got a slightly newer version of `prettier` when generating the `yarn.lock`. I'll add a `.prettierignore` and fix the formatting.